tests/TypeParser/JMSTypeParserTest.php

```diff
@@ -178,30 +178,30 @@ public function provideDateTimeTypeCases(): iterable
             'DateTimeImmutable|null',
             null,
             null,
-            'Y-m-d',
+            ['Y-m-d'],
         ];
 
         yield [
             'DateTimeImmutable<\'Y-m-d H:i:s\', \'Europe/Zurich\', \'Y-m-d\'>',
             'DateTimeImmutable|null',
             'Y-m-d H:i:s',
             'Europe/Zurich',
-            'Y-m-d',
+            ['Y-m-d'],
         ];
     }
 
     /**
      * @dataProvider provideDateTimeTypeCases
      */
-    public function testDateTimeType(string $rawType, string $expectedType, ?string $expectedFormat, ?string $expectedZone, ?string $expectedDeserializeFormat): void
+    public function testDateTimeType(string $rawType, string $expectedType, ?string $expectedFormat, ?string $expectedZone, ?array $expectedDeserializeFormats): void
```

Comment:
same here, i'd keep the single format as well for now and only remove it in version 2.